src/layout/Body.js

```diff
@@ -1,21 +1,20 @@
 import React, { useState, useEffect, useContext } from 'react';
-import {
-    Button,
-    Card,
-    CardBody,
-    CardHeader,
-    CardFooter,
-    Grid,
-    GridItem,
-    Stack,
-    StackItem,
-    Title,
-    PageSection,
-    Tooltip,
-    TooltipPosition
-} from '@patternfly/react-core';
-
-import { ArrowRightIcon, BinocularsIcon, CodeIcon, OutlinedEyeIcon } from '@patternfly/react-icons';
+import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
```

Comment:
Shouldn't this be solved by `transform-imports` plugin?

Reply:
Yeah, i have forgotten that it's used in this package as well.